addons/dark-www/experimental_scratchwww.css

```diff
@@ -226,6 +227,8 @@ body:not(.sa-body-editor) .button.white,
 .tabs button:hover,
 .tabs button.active:hover,
 .grid .thumbnail .thumbnail-info .thumbnail-title .thumbnail-creator a,
+.youtbe-video-button .video-info .video-title,
```

Comment:
It's `youtbe`? 😄

Reply:
Apparently. I copied the class name and didn't even notice that.